### ISSUE TITLE
Internal sealed class are exported

### ISSUE THREAD

**Issue by cl3m** [icerockdev/moko-kswift#35]
Internal sealed class are exported by kswift but won't be available in the library. Swift compilation failed because it can not find types.

```
internal sealed class Response<T> {
    data class Success<T>(val result: T) : Response<T>()
    data class Failure<T>(val error: String?) : Response<T>()
}
```

**Comment by cl3m** [comment]
Internal sealed class can be hidden with the @KSwiftExclude annotation but it does not work for the sub class.
```
sealed class Test {
    data class Exported(val value: Int) : Test()
    @KSwiftExclude // does not work, nor internal keyword
    internal data class Hidden(val value: Int) : Test()
}
```
Such class will make swift fail compilation as well.

**Comment by Alex009** [comment]
will be available in 0.4.0

**Comment by lammertw** [comment]
Apparently this also doesn't work for sealed classes under another internal class. Example:
```
internal interface MyInterface {
    sealed class Test {} // still gets exported
}
```

Not sure if there is any way to fix that.
